server/routes/journeys/update-plan/controller.ts

```diff
@@ -26,7 +57,7 @@ export class UpdatePlanController extends BaseJourneyController {
       ...getNonUndefinedProp(plan, 'caseManager'),
       ...getNonUndefinedProp(plan, 'reasonForPlan'),
       ...getNonUndefinedProp(plan, 'firstCaseReviewDate'),
-      identifiedNeeds: plan.identifiedNeeds.sort(identifiedNeedSorter).map(need => ({
+      identifiedNeeds: (plan.identifiedNeeds || []).sort(identifiedNeedSorter).map(need => ({
```

Comment:
can `plan.identifiedNeeds` be nullish?
i thought backend would at least return an empty array? (not sure)

Reply:
Yeah it was crashing on one of the csip records I was trying earlier